bibtex.y

```diff
@@ -60,9 +61,9 @@ preambleentry : tATSIGN tPREAMBLE tLBRACE longstring tRBRACE { $$ = $4 }
               ;
 
 longstring :                  tIDENT     { $$ = NewBibConst($1) }
-           |                  tBAREIDENT { $$ = bib.GetStringVar($1) }
+           |                  tBAREIDENT { v,err := bib.GetStringVar($1); if err != nil { bibtexlex.Error(fmt.Sprintf("%v",err)) } else {$$ = v} }
```

Comment:
whitespaces and I'd probably use the `Error()` to convert to error to string than `fmt` it.

```suggestion
           |                  tBAREIDENT { v, err := bib.GetStringVar($1); if err != nil { bibtexlex.Error(err.Error()) } else {$$ = v} }
```